### ISSUE TITLE
Status bar focus movement not complete

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#199307]
Testing #199199

1. Install azure account extension. Login
2. F6 to focus on status bar. Press right arrow to move focus. The focus is not moved across the whole status bar, it stops half way through (after problems entry) 🐛 

The issue is probably that we use two action bars in the status bar, and we are not properly handling focus transition via keyboard between the two action bars.

![Screenshot 2023-11-28 at 12 28 53](https://github.com/microsoft/vscode/assets/1926584/4d0ec52f-591e-468b-a0b9-39f783035f30)

**Comment by meganrogge** [comment]
These are all in one toolbar, so it should just work. @bpasero, the logs are showing that for some reason, the problems status bar item is `hidden`, which is why focus is getting stuck there. 

Do you know why that would be the case? 

![Screenshot 2023-11-28 at 2 37 58 PM](https://github.com/microsoft/vscode/assets/29464607/ed667788-a999-439d-ad51-b7518d8715f7)

I cannot explain this based on my changes https://github.com/microsoft/vscode/commit/a35724eb8c06e26ac597ad8d5b012757b7f6b746#diff-f41a6ab4f3b54529093781a2f325a672bcc531a48cb642452ac8a45767c43795

**Comment by meganrogge** [comment]
Oh, I wonder if the problems changes from @justschen can explain this. https://github.com/microsoft/vscode/pull/198719

**Comment by meganrogge** [comment]
I see that `problems.showCurrentInStatus` is off by default

**Comment by justschen** [comment]
Probably related to this:
https://github.com/microsoft/vscode/blob/1501e97f5288ee7ff32a4dce015c4faed8fe6ddd/src/vs/workbench/contrib/markers/browser/markers.contribution.ts#L571

Added this as part of the experimental setting related to turning off problems -> the reason there is and extra empty statusbar item is because I still return an item, even when it's false. Will push a fix for this!
